File: pfff/tests/js/naming/vardef_vs_use.js
// global
var x = 1;
foo(x);

function foo(a) {
    // local
    var y = 1;

    bar(y);
    bar(x);
    bar(a);
}

function test2() {
  // the addition of the local y in foo should not be visible here
  bar(y)
}

File: semgrep-core/tests/naming/go/vardef_vs_use.go
package Foo

// global
var x = 1

func foo(a int) {
  //local
  var y = 1

  bar(y)
  bar(x)
  bar(a)

}

func test2() {
 // the addition of the local y in foo should not be visible here
 bar(y)
}
